packages/react/src/FilterButtonList.js

```diff
@@ -10,6 +10,7 @@ import styles from './styles/index.js'
 import { newNodeFromField } from './utils/search.js'
 import { unusedOptions } from './FilterAdder.js'
 import { fieldsToOptions } from './utils/fields.js'
+import { displayLabelFn } from './utils/format.js'
```

Comment:
I think these `displayLabelFn` changes snuck in from another branch of yours

Reply:
Possibly, I will remove it. thanks for catching this.

Reply:
There is way more than just that, it all made it in, I will clean up this PR